pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: dash_kit_uikit
-description: A Flutter plugin to demonstrate the set of widgets created for an app with available widget states. This plugin may be used either for widget testing or to for getting acquainted with widgets used in project.
-version: 3.0.0
+description: A Flutter plugin to demonstrate the set of common widgets used in APP (for dev introduction and testing)
+version: 3.0.1
```

Comment:
Added such version in changelog as well, please

Reply:
Updated